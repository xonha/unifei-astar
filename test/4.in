9
000000000
00000F000
000F0000T
T0T000T00
F0T000000
T000T0000
000000000
000000000
00000T000
