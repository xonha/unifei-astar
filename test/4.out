Tomou -70 de dano! (quanto mais negativo melhor)
